src/main/java/nextstep/subway/domain/path/DurationPathStrategy.java

```diff
@@ -0,0 +1,18 @@
+package nextstep.subway.domain.path;
+
+import nextstep.subway.domain.Section;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
아래와 같이 하시면, PathService 에서 `Map`으로 주입받아도 의도와 같이 동작합니다 😄 

```suggestion
@Component("DURATION")
```

Reply:
넵! 감사합니다!